libraries/core_libs/network/src/network.cpp

```diff
@@ -13,32 +13,33 @@
 
 namespace taraxa {
 
-Network::Network(NetworkConfig const &config, const h256 &genesis_hash,
+Network::Network(const FullNodeConfig &config, const h256 &genesis_hash,
```

Comment:
sorry I missed why did you change this? I did not see where it is used

Reply:
because in multiple packet handlers I am using "delegation_delay" to know if it even makes sense to process e.g. votes and validate them against dpos contract